package.json

```diff
@@ -110,13 +111,15 @@
     "lint-staged": "^12.1.2",
     "load-grunt-tasks": "^5.1.0",
     "npm-run-all": "^4.1.1",
-    "octonode": "^0.9.4",
+    "octonode": "^0.10.2",
+    "postcss": "^7.0.39",
```

Comment:
is it a direct dev dependency or the resolution would be enough?

Reply:
For some reason, postcss was not added to node_modules, so a grunt script requiring it failed.

Reply:
I cannot remove postcss as a direct dependency now without breaking at least the `build` script